apycula/chipdb.py

```diff
@@ -165,6 +168,10 @@ def xls_pinout(family):
         return {
             "GW1N-4": get_pins("GW1N-4"),
         }
+    elif family == "GW1NS-2":
+        return {
+            "GW1NS-2C": get_pins("GW1NS-2C"),
```

Comment:
No `C` here, right?

Reply:
Or rather... I think we should add both the C and non-C variants. They can be supported by the same chipdb, it's only the pins that get added.

Reply:
Like with the GW1N-9 and GW1NR-9, same bitstream, different pinouts.

Reply:
There is no GW1NS-2C directory in IDE/share so yes need to use GW1NS-2
Pushed GW1NS-2 variant for pinout